distro/jbossas7/modules/src/main/modules/org/camunda/bpm/jboss/camunda-jboss-subsystem/main/module.xml

```diff
@@ -39,6 +39,9 @@
 
     <module name="org.camunda.bpm.camunda-engine-plugins" optional="true" />
 
+    <module name="org.camunda.connect.camunda-connect-core" />
+    <module name="org.camunda.connect.camunda-connect-http-client" services="import" />
```

Comment:
This change must also be made in the Wildfly subsystem, right?

Reply:
as discussed, it's applied.